packages/@aws-cdk/aws-pipes-targets-alpha/lib/eventBridge.ts

```diff
@@ -0,0 +1,145 @@
+import { IInputTransformation, IPipe, ITarget, TargetConfig } from '@aws-cdk/aws-pipes-alpha';
+import { IEventBus } from 'aws-cdk-lib/aws-events';
+import { IRole } from 'aws-cdk-lib/aws-iam';
+
+/**
+ * EventBridge target properties.
+ */
+export interface EventBridgeTargetParameters {
+  /**
+   * The input transformation to apply to the message before sending it to the target.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetparameters.html#cfn-pipes-pipe-pipetargetparameters-inputtemplate
+   * @default none
+   */
+  readonly inputTransformation?: IInputTransformation;
+
+  /**
+   * A free-form string, with a maximum of 128 characters, used to decide what fields to expect in the event detail.
+   *
+   * Minimum: 1
+   * Maximum: 128
```

Comment:
As far as I can tell, the CDK doesn't really JSDoc string length constraints like Cfn does:

https://github.com/aws/aws-cdk/blob/185f93d40a643b651be9c6668bc2b0772d40fcef/packages/aws-cdk-lib/aws-cloudfront/lib/cache-policy.ts#L20-L25

https://github.com/aws/aws-cdk/blob/185f93d40a643b651be9c6668bc2b0772d40fcef/packages/aws-cdk-lib/aws-cloudfront/lib/cache-policy.ts#L138-L140

```suggestion
   * A free-form string used to decide what fields to expect in the event detail.
```